daft/expressions/expressions.py

```diff
@@ -265,7 +265,7 @@ def from_expression(cls: type[SomeExpressionNamespace], expr: Expression) -> Som
 
 class ExpressionFloatNamespace(ExpressionNamespace):
     def is_nan(self) -> Expression:
-        raise NotImplementedError("[RUST-INT] Implement float is_nan expression")
+        return Expression._from_pyexpr(self._expr.is_nan())
```

Comment:
@jaychia I also noticed that we don't have `float.fill_nan()`, lmk if you'd like that to be added in this PR!